corehq/apps/userreports/tasks.py

```diff
@@ -35,7 +35,6 @@ def rebuild_indicators(indicator_config_id):
             adapter.save(doc)
         except DataError as e:
             logging.exception('problem saving document {} to table. {}'.format(doc['_id'], e))
-    adapter.engine.dispose()
```

Comment:
Good question. Since we are using the engine directly we do need to dispose
it since db connections can't be reused across processes. What we should do
is use scoped sessions:
http://prschmid.blogspot.com/2013/04/using-sqlalchemy-with-celery-tasks.html